public/index.html

```diff
@@ -25,52 +25,144 @@
+    <link
+      rel="apple-touch-icon"
+      sizes="120x120"
+      href="/assets/favicon-120.png"
+    />
+    <link rel="apple-touch-icon" sizes="76x76" href="/assets/favicon-76.png" />
+    <link
+      rel="apple-touch-icon"
+      sizes="152x152"
+      href="/assets/favicon-152.png"
+    />
+    <link
+      rel="apple-touch-icon"
+      sizes="180x180"
+      href="/assets/favicon-180.png"
+    />
+    <meta name="msapplication-TileColor" content="#C4C4C4" />
+    <meta name="msapplication-TileImage" content="/assets/favicon-144.png" />
+    <meta name="msapplication-config" content="/browserconfig.xml" />
 
     <!-- Load main local JS libraries -->
     <script type="module" src="js/main.js"></script>
   </head>
 
   <body>
+    <header>
+      <h3><strong>Hands Down!</strong></h3>
+    </header>
     <div class="container">
       <div class="row">
-        <h2>Hands-Down!</h2>
-      </div>
-      <div class="row">
-        <div class="eight columns">
+        <div class="one-third columns">
           Collection:
-          <button id="collection-state-btn" class="button-primary">Off</button>
-          Label: <button id="label-btn">True</button>
+          <button id="collection-state-btn" class="button-primary button-off">
+            Off
+          </button>
+        </div>
+        <div class="one-third columns">
+          <h6>
+            <strong><span id="collection-txt">0</span> Data Points</strong>
+          </h6>
+        </div>
+        <div class="one-third columns">
+          Label: <button id="label-btn" class="button-secondary">True</button>
         </div>
       </div>
       <div class="row"></div>
       <div class="row">
-        <div class="eight columns">
+        <div class="one-third columns">
           Inference:
           <button id="inference-state-btn" class="button-primary">On</button>
-          Inference: <span id="inference-txt"></span>
         </div>
-      </div>
-      <div class="row">
-        <div class="eight columns">
-          Collected <span id="collection-txt">0</span> Data Points
+        <div class="one-third columns">
+          <h6>
+            <strong
+              ><span class="mdi mdi-hand-right"></span> &nbsp;<span
+                id="inference-txt"
+              ></span
+            ></strong>
+          </h6>
+        </div>
+        <div class="one-third columns">
           <button id="export-btn">Export Data</button>
         </div>
       </div>
       <div class="row">
+        <div class="one column"></div>
         <div class="canvas-wrapper">
           <video id="video" style="display: none;"></video>
-          <div class="row">
-            <canvas
-              id="output"
-              width="640"
-              height="480"
-              style="border: 1px solid black;"
-            ></canvas>
-          </div>
+          <!-- Changing the size here doesn't change much as the drawing changes the size again -->
```

Comment:
If the width/height declarations don't do anything, should we take them out?

Reply:
Yeah, I although I wonder if there's a way to get the canvas to remain at the specified size while the drawing adapts to that without being distorted or mirrored (which happens when you force the `canvas`'s dimensions on the HTML side). Or if the video API can accept more responsive dimensions.